GitUI/CommandsDialogs/BrowseDialog/FormUpdates.cs

```diff
@@ -177,6 +174,45 @@ private async void btnDownloadNow_Click(object sender, EventArgs e)
             {
             }
         }
+
+        private void linkDirectDownload_LinkClicked(object sender, LinkLabelLinkClickedEventArgs e)
+        {
+            LaunchURL(LaunchType.DirectDownload);
+        }
+
+        private void LaunchURL(LaunchType launchType)
+        {
+            switch (launchType)
+            {
+                case LaunchType.ChangeLog:
+                    Process.Start("https://github.com/gitextensions/gitextensions/blob/master/GitUI/Resources/ChangeLog.md");
+                    break;
+                case LaunchType.DirectDownload:
+                    Process.Start(UpdateUrl);
+                    break;
+            }
+        }
+
+        protected override bool ProcessCmdKey(ref Message msg, Keys keyData)
+        {
+            // We need to override ProcessCmdKey as mnemonics on labels do not behave the same as buttons
+            if (keyData == (Keys.Alt | Keys.L))
+            {
+                LaunchURL(LaunchType.ChangeLog);
+            }
+            else if (keyData == (Keys.Alt | Keys.D))
+            {
+                LaunchURL(LaunchType.DirectDownload);
+            }
+
+            return base.ProcessCmdKey(ref msg, keyData);
+        }
+    }
+
+    public enum LaunchType
```

Comment:
Does is have to be `public`?

Reply:
I just cloned the ReleaseType enum. Declaring as private "Error	CS1527	Elements defined in a namespace cannot be explicitly declared as private, protected, protected internal, or private protected".

Internal works, are you happy for me to set this to internal?

Reply:
Set to internal on current commit

Reply:
That's fine, thanks.